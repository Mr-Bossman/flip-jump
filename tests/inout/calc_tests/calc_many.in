1
2
-6
5 + 7
 1  - 2
13+2

7
0
-2+2
-2-2


g
0x
x9
xbabe
q
