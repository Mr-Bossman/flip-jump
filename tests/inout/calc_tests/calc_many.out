>  
1
>  
2
>  
-6
>  
12
>  
-1
>  
15
>  
>  
7
>  
0
>  
0
>  
-4
>  
>  
>  
Error!
>  
Error!
>  
0x9
>  
0xBABE
>  
